editor.planx.uk/src/@planx/components/Pay/Public/Confirm.tsx

```diff
@@ -81,19 +81,25 @@ const PayBody: React.FC<PayBodyProps> = (props) => (
             size="large"
             onClick={props.onConfirm}
           >
-            {props.buttonTitle || "Pay using GOV.UK Pay"}
+            {props.buttonTitle || "Pay now using GOV.UK Pay"}
           </Button>
           {props.showInviteToPay && (
             <>
-              <Typography variant="body2">or</Typography>
-              <Link
-                component="button"
+              <Button
+                variant="contained"
+                color="secondary"
+                style={{ borderBottom: `solid 2px lightgrey` }}
```

Comment:
This is perfect here for now for a single implementation, but I think going forward we can define a "primary" and "secondary" button variant on the theme - https://mui.com/material-ui/customization/theme-components/#theme-style-overrides